torch/package/package_importer.py

```diff
@@ -340,17 +357,26 @@ def _make_module(
     def _load_module(self, name: str, parent: str):
         cur: _PathNode = self.root
         for atom in name.split("."):
-            if not isinstance(cur, _PackageNode) or atom not in cur.children:
+            if isinstance(cur, _SelectiveInternNode):
```

Comment:
Can you explain this bit? I am reading it as: when we see an unknown child of a selective intern node, mark it extern.

This is confusing to me on two levels:
1. There should not be any missing children. We should expect the module tree to be fully populated after PackageImporter initialization. It is an error for there to be a missing node (hence the ModuleNotFoundError below).
2. Why are we marking the missing node as extern? That implies that for, `torch.fb.unknown`, we would be going selective extern -> selective intern -> extern, which we never want to do.